snap/snapcraft.yaml

```diff
@@ -0,0 +1,80 @@
+name: edgex-cli
+base: core18
+type: app
+adopt-info: version
+summary: A command-line client for EdgeX Foundry
+description: |
+  A command-line client for EdgeX Foundry.
+
+grade: stable
+confinement: strict
+
+apps:
+  edgex-cli:
+    adapter: none
+    command: bin/edgex-cli
+    command-chain:
+      - bin/edgex-cli-wrapper.sh
+    plugs: [home, network]
+
+parts:
+  version:
+    plugin: nil
+    # the source dir is unrelated to this part and is used since it
+    # changes rarely and therefore will not trigger a new pull
+    source: snap/local
+    override-pull: |
+      cd $SNAPCRAFT_PROJECT_DIR
+      if [ -f VERSION ]; then
+        PROJECT_VERSION=$(cat VERSION)
+      else
+        PROJECT_VERSION=local-dev
+      fi
+      GIT_REVISION=$(git rev-parse --short HEAD)
```

Comment:
No harm in setting the GIT_REVISION variable, but it's not needed anymore as we we don't use the git revision in the version string.

Reply:
@siggiskulason Thanks for the review! Just removed the GIT_REVISION line and re-pushed.